common/lib/dependabot/pull_request_creator/branch_namer/solo_strategy.rb

```diff
@@ -221,6 +227,30 @@ def dependency_digest
             T.nilable(String)
           )
         end
+
+        sig do
+          params(requirements: T.nilable(T::Array[T::Hash[Symbol,
+                                                          T.untyped]])).returns([T.nilable(String), T.nilable(String)])
```

Comment:
This formatting is a crime against humanity 😂

Reply:
Blame that on vscode 🤷